pom.xml

```diff
@@ -230,8 +230,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.7</source>
+                    <target>1.7</target>
```

Comment:
I would go even further:
1) Update parent POM to 2.x latest
2) Update Core requirement to 1.609.x or above
3) Get rid of this section, you will get Java 7 by default